Source/SmallBasic.Editor/Components/Pages/Edit/LibraryBody.scss

```diff
@@ -112,9 +113,34 @@ library-body {
                 name {
                     margin-left: 10px;
                 }
+
+                error-corner-ribbon {
```

Comment:
Can we make `error-corner-ribbon` the parent without specifying the color, and then have `error-corner-ribbon-blue` and `error-corner-ribbon-gray` for readability?